httpie/compat.py

```
@@ -0,0 +1,173 @@
+            curr = end[1]
+            while curr is not end:
+                yield curr[0]
+                curr = curr[1]
+
+        def popitem(self, last=True):
+            if not self:
+                raise KeyError('dictionary is empty')
+            if last:
+                key = reversed(self).next()
+            else:
+                key = iter(self).next()
```

Ruff (0.8.2)

Use ternary operator `key = reversed(self).next() if last else iter(self).next()` instead of `if`-`else`-block

Replace `if`-`else`-block with `key = reversed(self).next() if last else iter(self).next()`

(SIM108)